.github/workflows/upload.yml

```diff
@@ -29,20 +38,19 @@ jobs:
           mv mrmustard src
 
       - name: Install only test dependencies
-        run: uv sync --inexact --no-install-project --all-extras
+        run: uv sync --only-dev --inexact --no-install-project
 
       - name: Run tests
         run: uv run --no-project pytest tests -p no:warnings --tb=native --backend=tensorflow
 
-      - name: Publish
-        uses: pypa/gh-action-pypi-publish@release/v1
-        with:
-          user: __token__
-          password: ${{ secrets.PIPY_TOKEN }}
+      # - name: Publish
+      #   uses: pypa/gh-action-pypi-publish@release/v1
+      #   with:
+      #     user: __token__
+      #     password: ${{ secrets.PIPY_TOKEN }}
 
       - name: Install MrMustard from pypi
-        run: uv pip install mrmustard
+        run: uv pip install --pre --upgrade mrmustard
```

Comment:
The `--pre` flag will ensure we can get the pre-release version (see https://pip.pypa.io/en/stable/cli/pip_install/#pre-release-versions).

Reply:
Won't `--prerelease=allow` install pre-release versions for all dependencies? Note that I did test this both locally and at https://github.com/XanaduAI/MrMustard/actions/runs/16473307388/job/46568016618

Reply:
Better yet, I could clear the virtual environment?

Reply:
Hmm... `--no-cache-dir` is maybe the best call, actually.